packages/drivers/odsp-driver/src/test/createNewUtilsTests.spec.ts

```diff
@@ -165,7 +165,12 @@ describe("Create New Utils Tests", () => {
 			{ "x-fluid-epoch": "epoch1" },
 		);
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-		const snapshot = await epochTracker.get(createCacheSnapshotKey(odspResolvedUrl));
+		const snapshot = await epochTracker.get(createCacheSnapshotKey(odspResolvedUrl, false));
```

Comment:
It would be nice to see all the different values tested at least in one unit test.

`false`, `undefined`, `true`.

It would be nice to see it in e2e tests, maybe as more of an addition to a test (we can kind of test multiple different features at once with an e2e test). As we are testing a bunch of different features. I don't think it's a requirement for this, but if we already do, then feel free to ignore.